cfme/containers/provider/__init__.py

```diff
@@ -440,24 +439,23 @@ def step(self):
 
 class ContainerObjectAllBaseView(ProvidersView):
     """Base class for container object All view.
-    TITLE_TEXT should be defined in child."""
-
+    TITLE_TEXT should be defined in child.
+    """
+    summary = Text('//div[@id="main-content"]//h1')
     policy = Dropdown('Policy')
     download = Dropdown('Download')
 
     @property
     def table(self):
         return self.entities.elements
 
+    @property
     def title(self):
-        if not hasattr(self, 'TITLE_TEXT'):
-            raise Exception('TITLE_TEXT is not defined for destination {} ("All").'
-                            .format(self.__class__.__name__))
-        return Text('//h1[normalize-space(.) = {}]'.format(quote(self.TITLE_TEXT)))
+        return self.summary
 
     @property
     def is_displayed(self):
-        return self.title.is_displayed
+        return self.summary.text == self.TITLE_TEXT
```

Comment:
**REQUIRED:** Is ```self.title``` still needed now as all it does is return summary. Wouldn't it be better to return ```self.summary.text``` and then you can just direct compare ```self.title == self.TITLE_TEXT```

Reply:
Well, I just thought to keep this as it is for backward compatibility, but now I see that it's not really in use anywhere so maybe worth to just delete `def title` (?).

Reply:
Yeh if it's not used, lets delete it, particularly in this case as I think is is a little confusingly named.